Userland/Libraries/LibVideo/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(SOURCES
+    DecoderError.h
```

Comment:
Header files should not be added to the SOURCES set. This basically instructs CMake to build these header files, which is not something we want.

Might be a good idea to also remove all the other headers in this file.

Reply:
Actually it doesn't. IDEs like QtCreator use headers in the sources list (or target_headers or w/e) to populate the project.

but correct, we don't normally put headers in SOURCES.

Reply:
Yeah, here I made the mistake of following what was already in the file.. I've added a first commit to remove the existing ones and fixed the added one.